tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -971,9 +978,10 @@ class RKeyBlob : public TKey {
       Create(nbytes);
       *seekKey = fSeekKey;
    }
+
+   ClassDefInline(RKeyBlob, 0)
```

Comment:
I put version=0 because in my understanding this class never gets serialized by the ROOT I/O